index.js

```diff
@@ -209,6 +209,9 @@ const ScrollableTabView = React.createClass({
     if (this.props.tabBarUnderlineColor) {
       tabBarProps.underlineColor = this.props.tabBarUnderlineColor;
     }
+    if (this.props.tabBarUnderlineHeight) {
+      tabBarProps.underlineHeight = this.props.tabBarUnderlineHeight;
+    }
```

Comment:
I don't want to pass property through ```index.js``` in future I will remove properties like ```tabBarUnderlineColor``` from there